skyrim-platform/src/platform_se/CMakeLists.txt

```diff
@@ -168,11 +167,21 @@ if(NOT "${SKIP_SKYRIM_PLATFORM_BUILDING}")
   add_custom_target(RestartGame ALL COMMAND ${CMAKE_COMMAND} -E touch ${CMAKE_CURRENT_BINARY_DIR}/touch_$<CONFIG>)
   add_dependencies(RestartGame ${DEPENDENCIES_FOR_CUSTOM_TARGETS})
 
-  add_custom_target(codegen ALL
-    COMMAND node ${PROJECT_SOURCE_DIR}/src/platform_se/codegen/index.js
-    WORKING_DIRECTORY ${wdir}
+  add_executable(TSConverter ${CMAKE_SOURCE_DIR}/skyrim-platform/src/platform_se/codegen/TSConverter.cpp)
+  apply_default_settings(TARGETS TSConverter)
+  set_target_properties(TSConverter PROPERTIES
+      CXX_STANDARD 20
+    )
+  list(APPEND VCPKG_DEPENDENT TSConverter)
+  
+  set(CONVERT_FILES_DIR ${CMAKE_SOURCE_DIR}/skyrim-platform/src/platform_se/codegen/convert-files/)
```

Comment:
Maybe we should use `PROJECT_SOURCE_DIR` or `CMAKE_CURRENT_LIST_DIR` instead. This would allow to prevent hardcoding parts of the path